test-data/unit/check-classes.test

```diff
@@ -2051,6 +2051,16 @@ class X: pass
 foo(X)
 [out]
 
+[case testTypeUsingTypeCTypeAnyMember]
+from typing import Type, Any
+def foo(arg: Type[Any]):
+    x = arg.member_name
```

Comment:
@dmoisset You could update one of the builtins fixtures  by adding the correct type signature from typeshed, and use it with e.g. ``[builtins fixtures/list.pyi]`` at the end of test case.